src/utilities/analytics.ts

```diff
@@ -0,0 +1,72 @@
+
+/**************************************************************
+ * This file exports a few interfaces and a single function
+ * `logAnalyticsEvent` which is a small wrapper around
+ * Google Analytics service custom event logging.
+ *************************************************************/
+
+import { TeacherTipType } from "../types";
+
+export enum AnalyiticsActionType {
```

Comment:
Typo: `AnalyiticsActionType` => `AnalyticsActionType` which is then propagated to clients.

Reply:
Thanks @kswenson -- your comments addressed in e06eb15